src/forms/add-workflow.schema.js

```diff
@@ -19,7 +19,7 @@ const addWorkflowSchema = (intl) => ({
     fields: [{
       name: 'general-information',
       showTitle: true,
-      customTitle: <Title size="md"> { <FormattedMessage id="enter-info" defaultMessage="Enter your information" /> } </Title>,
+      customTitle: <Title headingLevel="h3" size="md"> { <FormattedMessage id="enter-info" defaultMessage="Enter your information" /> } </Title>,
```

Comment:
The default value for Title used to be `h1` I am guessing the sizes did not match?